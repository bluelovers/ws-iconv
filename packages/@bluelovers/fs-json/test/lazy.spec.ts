import { stringifyJSON } from '../index';

test(`stringify`, () =>
{

	let actual = stringifyJSON({});

	expect(actual).toMatch(/\n$/);
	expect(actual).toMatchSnapshot();

});

