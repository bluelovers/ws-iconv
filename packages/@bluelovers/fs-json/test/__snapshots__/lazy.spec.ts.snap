// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stringify 1`] = `
"{}
"
`;
